src/sources/host_metrics.rs

```diff
@@ -692,6 +693,33 @@ fn add_collector(collector: &str, mut metrics: Vec<Metric>) -> Vec<Metric> {
     metrics
 }
 
+pub fn init_roots() {
+    #[cfg(target_os = "linux")]
+    {
+        match std::env::var_os("PROCFS_ROOT") {
+            Some(procfs_root) => {
+                info!(
+                    "PROCFS_ROOT is set in envvars. Using {:?} for procfs.",
+                    procfs_root
+                );
```

Comment:
I think there was an overhaul of logs in which we moved things to structured logging form from the formatting form as much as possible. Not sure if it's a strict rule now to use the structured form, cc @jamtur01

```suggestion
                info!(
                    message = "PROCFS_ROOT is set in envvars. Using custom for procfs.",
                    ?procfs_root
                );
```
It would look something like this if we were to used structured form.

The same should probably be applied to the logs below.